conda/cli/install.py

```diff
@@ -130,11 +130,27 @@ def get_revision(arg, json=False):
         CondaValueError("expected revision number, not: '%s'" % arg, json)
 
 
+def check_non_admin():
+    if not context.non_admin_enabled:
+        if on_win:
+            if on_win:
```

Comment:
Oops.  I guess I shouldn't have put the "Ready for Review" tag on yet since I haven't written any tests.  Thanks for the catch.